backend/common/src/main/java/ai/verta/modeldb/common/futures/FutureJdbi.java

```diff
@@ -16,15 +16,36 @@ public FutureJdbi(Jdbi jdbi, Executor executor) {
     this.jdbi = jdbi;
   }
 
+  @FunctionalInterface
+  public interface SupplierWithException<R, T extends Exception> {
+    R get() throws T;
+  }
+
+  @FunctionalInterface
+  public interface RunnableWithException<T extends Exception> {
+    void run() throws T;
+  }
```

Comment:
These don't have to be public, right? Them seem like internal implementation details that are never actually exposed to the user.

Reply:
yes, changed to private